src/transformers/tokenization_utils_fast.py

```diff
@@ -699,6 +700,8 @@ def train_new_from_iterator(
             kwargs["end_of_word_suffix"] = tokenizer_json["model"]["end_of_word_suffix"]
         if tokenizer_json["model"]["type"] == "Unigram" and unk_token is not None:
             kwargs["unk_token"] = unk_token
+        if tokenizer_json["pre_tokenizer"]["type"] == "ByteLevel":
+            kwargs["initial_alphabet"] = pre_tokenizers_fast.ByteLevel.alphabet()
```

Comment:
At some point everything in this should be ported directly within `tokenizers`.

Information flow from the `Tokenizer` to the `trainer` is a long standing issue (some options are recoverable, some are not, but it's inconsistent currently)